vmodules/util/interfaces/BotConfig.ts

```diff
@@ -17,5 +29,69 @@ export default interface BotConfig {
   },
   api: {
     port: number
-  }
+  },
+  colors: {
+    default: ColorResolvable
+  },
+  metadata: {
+    version: number,
+    updated: number
+  },
+  lang: 'en_us' | 'en_test',
+  default_lang: string,
+  automation: {
+    auto_cmd: any[],
+    auto_react: any[],
+    auto_reply: any[],
+    auto_mod: any[]
+  },
+  commands: {
+    owo: boolean,
+    prefixes: string[],
+    hidden: Command[], // TODO: check with Jack on this
```

Comment:
Are these commands that are hidden? or command names as with the line below?

Reply:
names only

Reply:
i've resolved this locally and will push soon